examples/inference/smc_random_walk/NICER_J0614/submit.sh

```
@@ -0,0 +1,28 @@
+#SBATCH --cpus-per-gpu=1
+#SBATCH --mem-per-gpu=40G
+#SBATCH --output="./outdir/log_smc_rw.out"
+#SBATCH --job-name="SMC-RW-J0614"
+
+now=$(date)
+echo "$now"
+
+# Loading modules
+# module load 2024
+# module load Python/3.10.4-GCCcore-11.3.0
+source /home/twouters2/projects/jester_review/jester/.venv/bin/activate
```

Shellcheck (0.11.0)

[info] Not following: /home/twouters2/projects/jester_review/jester/.venv/bin/activate was not specified as input (see shellcheck -x).

(SC1091)